master/buildbot/changes/bitbucket.py

```diff
@@ -89,11 +97,13 @@ def _getChanges(self):
                 "Bitbucket repository %s/%s, branch: %s" % (self.owner, self.slug, self.branch))
         url = "https://bitbucket.org/api/2.0/repositories/%s/%s/pullrequests" % (
             self.owner, self.slug)
-        return client.getPage(url, timeout=self.pollInterval)
+        return self._getPage(url)
```

Comment:
That is actually not the poll interval (that is set [here](https://github.com/fluendo/buildbot/pull/3/files/79866c74303de5373b8fbd54e76932fb2e22c25e#diff-b6e3d908323157092e3315ece3033181R66)), it's just using the same value for polling and for the connection timeout. I thought it would be better to leave the default.